data/actionsmap/yunohost.yml

```diff
@@ -240,132 +250,82 @@ user:
-                            extra:
-                                pattern: *pattern_username
 
-                ### user_permission_remove()
-                remove:
-                    action_help: Revoke access right to users and group
-                    api: PUT /users/permissions/<app>
+                ### user_permission_update()
+                update:
+                    action_help: Grant / remove permissions to groups or users
+                    api: POST /users/permissions/<permission>
                     arguments:
-                        app:
-                            help: Application to manage the permission
-                            nargs: "+"
-                        -p:
-                            full: --permission
-                            help: Name of permission (main by default)
-                            nargs: "*"
-                            metavar: PERMISSION
-                        -u:
-                            full: --username
-                            help: Username
+                        permission:
+                            help: Permission to manage (e.g. mail or nextcloud or wordpress.editors)
+                        -a:
+                            full: --add
+                            help: Group or usernames to grant this permission to
                             nargs: "*"
-                            metavar: USER
+                            metavar: GROUP_OR_USER
                             extra:
                                 pattern: *pattern_username
-                        -g:
-                            full: --group
-                            help: Group name
+                        -r:
+                            full: --remove
+                            help: Group or usernames revoke this permission from
                             nargs: "*"
-                            metavar: GROUP
+                            metavar: GROUP_OR_USER
                             extra:
                                 pattern: *pattern_username
 
-                ## user_permission_clear()
-                clear:
-                    action_help: Reset access rights for the app
+                ## user_permission_reset()
+                reset:
+                    action_help: Reset allowed groups to the default (all_users) for a given permission
                     api: DELETE /users/permissions/<app>
                     arguments:
-                        app:
-                            help: Application to manage the permission
-                            nargs: "+"
-                        -p:
-                            full: --permission
-                            help: Name of permission (main by default)
-                            nargs: "*"
-                            metavar: PERMISSION
+                        permission:
+                            help: Permission to manage (e.g. mail or nextcloud or wordpress.editors)
```

Comment:
Does we really want to share to user the concept separate the app name with the name of the permission with a dot ?

Reply:
Uuuh not sure to understand what you mean ?

Reply:
Before the technical name of the permission (app.permission) was not shared to the user. On the user side there was just an app with some permission.

Reply:
Hmmm idk, our conclusion with ljf was that it was unecessarily complicated to separate the two pieces of the permission and we should just talk about `wordpress.editors` as "the name of the permission" and that's it. 

`user_permission_list` now displays the name of the permission (e.g. wordpress.editors) with associated information with something like : 

```
wordpress.main:
     allowed: all_users
wordpress.editors:
     allowed: volunteers
mail.main:
     allowed: all_users
    
```

and if you want to edit it, you "just" have to `yunohost user permission update wordpress.editors --add foo --remove bar`

(The notion of `app.permission` was also confusing (though not a huge deal) because there's also `mail`, `xmpp` and other things to come which are not really apps)